tailwind.config.js

```diff
@@ -5,10 +5,55 @@ module.exports = {
     './node_modules/flowbite-react/**/*.js',
     './src/pages/**/*.{js,ts,jsx,tsx}',
     './src/components/**/*.{js,ts,jsx,tsx}',
-    './src/app/**/*.{js,ts,jsx,tsx}',
+    './src/apps/**/*.{js,ts,jsx,tsx}',
   ],
   theme: {
     extend: {
+      colors: {
+        primary: '#098383',
+        secondary: '#8C6F4C',
+        warn: '#DA483B',
+      },
+      fontFamily: {
+        "noto-sans": ['Noto Sans TC', 'sans-serif'],
```

Comment:
need lint fix, strings must use singlequote

Reply:
fixed, run lint is not working